examples/udp_client/linux_udp_client.cc

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/ /**
+ * @brief Simple Linux UDP client example.
+ * @file
+ *
+ * @note
+ * This is a minimal UDP client implementation, meant as an example of how to
+ * connect to a device and decode incoming data. It is not robust to network
+ * outages, socket reconnects, or other typical network errors. Production
+ * implementations must check for and handle these cases appropriately (by
+ * checking expected `errno` values, etc.).
+ ******************************************************************************/
+
+#include <cerrno>
+#include <cmath> // For lround()
+#include <csignal> // For signal()
+#include <cstdio> // For fprintf()
+#include <cstring> // For memcpy()
+#include <string> // For stoi() and strerror()
+
+#include <netdb.h> // For gethostbyname() and hostent
+#include <netinet/in.h> // For IPPROTO_* macros and htons()
+#include <sys/socket.h> // For socket support.
+#include <unistd.h> // For close()
+
+#include <iostream>
+#include <fstream>
+#include <arpa/inet.h>
+
+#include <point_one/fusion_engine/parsers/fusion_engine_framer.h>
+
+#include "../common/print_message.h"
+
+#define DEBUG_ON 0
+
+using namespace point_one::fusion_engine::examples;
+using namespace point_one::fusion_engine::parsers;
+
+static bool shutdown_pending_ = false;
+
+/******************************************************************************/
+void HandleSignal(int signal) {
+  if (signal == SIGINT || signal == SIGTERM) {
+    std::signal(signal, SIG_DFL);
+    shutdown_pending_ = true;
+  }
+}
+
+void * get_in_addr(struct sockaddr * sa) {
```

Comment:
Formatting of *s not quite right here and below - please run `clang-format` on the file to get all the style stuff.

Reply:
Add a `/***/` bar before this function, similar to the other functions.

Function names should be title case (`GetInAddr`).

Reply:
done.